portfolio/src/main/webapp/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+ <!DOCTYPE html>
```

Comment:
Remove extra space.

Reply:
It looks like the extra space is still here. Maybe you didn't push the last commit?

Reply:
Still looks like it's there.